src/react/features/connector/tools/buttons.js

```diff
@@ -1,6 +1,7 @@
 export const ADD_ROOM = 'ADD_ROOM'
+export const GENERATE_DIOGRAPH = 'GENERATE_DIOGRAPH'
```

Comment:
So we have a button called `GENERATE_DIOGRAPH` and a channel called `GENERATE_DIOGRAPH`? Is that on purpose?

I don't have any opinion about it, just pointing out :)

Reply:
Good point. GENERATE_DIOGRAPH_BUTTON at least. _CHANNEL for channels.